### ISSUE TITLE
SVG is rendered different size depending on devicePixelRatio

### ISSUE THREAD

**Issue by gahms** [dnfield/flutter_svg#99]
The same SVG is rendered in different size on different devices depending on the devicePixelRatio. E.g. on a device with devicePixelRatio = 2 a SVG with 
`width="26px" height="14px" viewBox="0 0 26 14"`
will be rendered at 52x28 points and on a device with devicePixelRatio = 3 it will render at 78x42 points.

This means that the same SVG will render at different physical sizes on the two devices.

It would seem natural to me that a SVG with size 26x14 will render at 26x14 points?

In other words I would not multiply the viewport size with devicePixelRatio in [svg.dart line 599](https://github.com/dnfield/flutter_svg/blob/84ffba65b397fa5b318b676fda5172f683939d75/lib/svg.dart#L599)

Did I miss something?

**Comment by dnfield** [comment]
This is by design.  For Flutter, everything is in logical rather than physical pixels.  Things start to look strange and inconsistent if this isn't respected.

That said, I know I'm not handling the case where size is specified quite right - see #62

**Comment by dnfield** [comment]
I think you could get what you want by setting the width and height properties on your widget to be divided by devicePixelRatio

**Comment by gahms** [comment]
Ok, but the SVG is not rendered as 26x14 pixels and it is not rendered as 26x14 points (~ logical pixels). 

In my case I am not specifying width nor height but want to render the SVG at its "natural" size. As far as I can see I have no way of extracting the SVG width and height to force the size of the Widget.

**Comment by gahms** [comment]
I will try once more to convince you :-) I load SVG files from the network and as such do not know their size beforehand. I want to show the SVG files at their natural size, i.e. the point size given by their width and height in the SVG file.

How do you propose I do that? 

I cannot wrap the SvgPicture in a widget and set the width and height of that because I do not know the width and height of the SvgPicture.

You say that everything in Flutter is logical pixels (points) but as SVG is a vector format, surely the sizes in SVG are in logical pixels (points) also. 

I cannot see how rendering the same SVG at different logical/point/physical size on different devices is ever what one wants. (physical size in e.g. mm, not pixels)

**Comment by dnfield** [comment]
I'm not really clear on what you mean by natural size.  Different devices will display the natural size differently, right?  

Sorry if I'm just being dense here - perhaps an illustration, or a PR with a suggested change to address this issue would help.

**Comment by gahms** [comment]
Thank you for your patience. I have created a demo project that illustrates the issue:
https://github.com/gahms/flutter_svg_size_bug

Also, I have created a pull request #113 with the changes I have made and is using in our app.

**Comment by dnfield** [comment]
This has shown to cause even more serious problems on a device where devicePixelRatio < 1.0

I'm going to try to get a fix for this in ASAP.

**Comment by dnfield** [comment]
Fixed in #131 (and #113)
